app/crowd/server/crowd.js

```diff
@@ -203,6 +204,13 @@ export class CROWD {
 				const response = self.crowdClient.searchSync('user', `email=" ${ email } "`);
 				if (!response || response.users.length === 0) {
 					logger.warn('Could not find user in CROWD with username or email:', crowd_username, email);
+					if (settings.get('CROWD_Remove_Orphaned_Users') === true) {
+						logger.info('Removing user');
+						Meteor.defer(function() {
+							deleteUser(user._id);
+						});
+						logger.info('Removed');
```

Comment:
this logger line will run before the user is removed due to using Meteor.defer. You can move it inside the deferred function so it only logs after the user is truly removed.